declarations/Amazon Mechanical Turk.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Amazon Mechanical Turk",
+  "documents": {
+    "Privacy Policy": {
+      "fetch": "https://www.amazon.com/gp/help/customer/display.html/ref=footer_privacy?ie=UTF8&nodeId=468496",
+      "select": [
+        ".cs-help-content"
+      ],
+      "remove": [
+        ".a-size-medium",
+        "#helpsearch",
+        ".cs-help-breadcrumb",
+        "#hmd-FeedbackBox > .a-box-inner"
```

This does not remove some invisible elements that get tracked:

> Thank you for your feedback.
> Please select what best describes the information:
> This information is confusing or wrong
> This isn't the information I was looking for
> I don't like this policy
> Submit
> Thanks! While we're unable to respond directly to your feedback, we'll use this information to improve our online Help.

This seems to fix it:
```suggestion
        ".a-box-group"
```

---

I did an even better selector that removes the need for `remove` selectors. And you could have done too through the [online contribution tool](https://contribute.opentermsarchive.org/en/service?destination=OpenTermsArchive%2Fp2b-compliance-declarations&documentType=Privacy%20Policy&name=Amazon%20Mechanical%20Turk&removedCss[]=.a-size-medium&removedCss[]=%23helpsearch&removedCss[]=.cs-help-breadcrumb&removedCss[]=%23hmd-FeedbackBox%20%3E%20.a-box-inner&selectedCss[]=.cs-help-content&url=https%3A%2F%2Fwww.amazon.com%2Fgp%2Fhelp%2Fcustomer%2Fdisplay.html%2Fref%3Dfooter_privacy%3Fie%3DUTF8%26nodeId%3D468496&expertMode=true)